lib/cea/mp4_cea_parser.js

```diff
@@ -236,7 +236,13 @@ shaka.cea.Mp4CeaParser = class {
           });
         }
       } else {
-        reader.skip(naluSize - 1);
+        try {
+          reader.skip(naluSize - 1);
+        } catch (e) {
+          if (e instanceof shaka.util.Error) {
+            break;
+          }
+        }
```

Comment:
I don't quite understand the logic here.  Some comments may be helpful.

If the error is a shaka.util.Error, you break out of the loop.  If it's some other kind of error, you continue.  I don't understand the purpose of either.

Reply:
you are right, although the function really only throws this error, it can be applied to everything.